bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,50 @@
+from discord.ext import commands
+from pathlib import Path
+from discord import Embed
+import random
+import json
+
+embeds = []
+
+# cog
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    def get_response(self) -> Embed:
+        return random.choice(embeds)
+
+    @commands.command(aliases=('save_the_earth',))
+    async def save_the_planet(self, ctx: commands.Context) -> None:
+        """Responds with a random tip on how to be ecofriendly and help our planet."""
+        await ctx.send(embed=self.get_response())
+
+def setup(bot: commands.Bot) -> None:
+    """save_the_planet Cog load."""
+        
+    with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding="utf8") as f:
+        responses = json.load(f)
+
+    # convert what's in the json to discord embed objects https://discord.com/developers/docs/resources/channel#embed-object
```

Comment:
A function with a descriptive name would make it a bit cleaner and remove the need for this comment.

Reply:
> Because of these your checks are failing
> 
> ```
> bot/exts/easter/save_the_planet.py:2:1: I100 Import statements are in the wrong order. 'from pathlib import Path' should be before 'from discord.ext import commands' and in a different group.
> bot/exts/easter/save_the_planet.py:2:1: I201 Missing newline between import groups. 'from pathlib import Path' is identified as Stdlib and 'from discord.ext import commands' is identified as Third Party.
> bot/exts/easter/save_the_planet.py:3:1: I201 Missing newline between import groups. 'from discord import Embed' is identified as Third Party and 'from pathlib import Path' is identified as Stdlib.
> bot/exts/easter/save_the_planet.py:4:1: I100 Import statements are in the wrong order. 'import random' should be before 'from discord import Embed' and in a different group.
> bot/exts/easter/save_the_planet.py:4:1: I201 Missing newline between import groups. 'import random' is identified as Stdlib and 'from discord import Embed' is identified as Third Party.
> bot/exts/easter/save_the_planet.py:5:1: I100 Import statements are in the wrong order. 'import json' should be before 'import random'
> bot/exts/easter/save_the_planet.py:9:1: E302 expected 2 blank lines, found 1
> bot/exts/easter/save_the_planet.py:15:1: D102 Missing docstring in public method
> bot/exts/easter/save_the_planet.py:23:1: E302 expected 2 blank lines, found 1
> bot/exts/easter/save_the_planet.py:24:1: D202 No blank lines allowed after function docstring
> bot/exts/easter/save_the_planet.py:29:121: E501 line too long (124 > 120 characters)
> ```

I think I've fixed that, but GitHub desktop was still telling me it needed to be fixed and I couldn't commit.